### DISCUSSION TITLE
获取Qzone功能被识别出来是登录空间了

### DISCUSSION THREAD

**Discussion by warnlop** [sun589/QQkey_Tool#21] [提问] [answered]
现在使用qzone 会被QQ识别出来是登录QQ空间，用不了了

**Comment by warnlop** [comment]
能让QQ识别是在登录问卷文档之类的吗？识别出来是QQ空间没人会登录的

**Comment by sun589** [comment, marked as answer]
识别出来是空间是必然的，简单解释一下：
QQ会识别登录后跳转的网站，然后根据网址获取对应的业务（即qzone）
如果替换掉跳转的网站的话 p_skey就不对应了，强制使用一个网站p_skey去使用其他网站的功能（比如登录并获取qun的p_skey,去使用获取说说），会验证p_skey后直接返回登录失败而无法操作，因此无法避免，Soooooooooooorryy:(

**Comment by warnlop** [comment]
因为这个网站手机QQ识别不出来吗



---- 回复的原邮件 ----
| 发件人 | ***@***.***> |
| 日期 | 2024年11月28日 19:20 |
| 收件人 | ***@***.***> |
| 抄送至 | ***@***.***>、State ***@***.***> |
| 主题 | Re: [sun589/QQkey_Tool] 获取Qzone功能被识别出来是登录空间了 (Discussion #21) |

识别出来是空间是必然的，简单解释一下：
QQ会识别登录后跳转的网站，然后根据网址获取对应的业务（即qzone）
如果替换掉跳转的网站的话 p_skey就不对应了，强制使用一个网站p_skey去使用其他网站的功能（比如登录并获取qun的p_skey,去使用获取说说），会验证p_skey后直接返回登录失败而无法操作，因此无法避免，Soooooooooooorryy:(

**Comment by sun589** [comment]
总的一句话：就是QQ看出来是在登录空间，是改不了的

**Comment by warnlop** [comment]
OK



---- 回复的原邮件 ----
| 发件人 | ***@***.***> |
| 日期 | 2024年11月28日 19:27 |
| 收件人 | ***@***.***> |
| 抄送至 | ***@***.***>、State ***@***.***> |
| 主题 | Re: [sun589/QQkey_Tool] 获取Qzone功能被识别出来是登录空间了 (Discussion #21) |

总的一句话：就是QQ看出来是在登录空间，是改不了的

**Comment by warnlop** [comment]
原来如此，那用qun.com为啥显示的是登录登录呢



---- 回复的原邮件 ----
| 发件人 | ***@***.***> |
| 日期 | 2024年11月28日 19:20 |
| 收件人 | ***@***.***> |
| 抄送至 | ***@***.***>、State ***@***.***> |
| 主题 | Re: [sun589/QQkey_Tool] 获取Qzone功能被识别出来是登录空间了 (Discussion #21) |

识别出来是空间是必然的，简单解释一下：
QQ会识别登录后跳转的网站，然后根据网址获取对应的业务（即qzone）
如果替换掉跳转的网站的话 p_skey就不对应了，强制使用一个网站p_skey去使用其他网站的功能（比如登录并获取qun的p_skey,去使用获取说说），会验证p_skey后直接返回登录失败而无法操作，因此无法避免，Soooooooooooorryy:(

**Comment by sun589** [comment]
不知道
